app/src/main/res/navigation/setup_activity_nav_graph.xml

```diff
@@ -18,7 +18,7 @@
       app:popUpToInclusive="true" />
     <action
       android:id="@+id/action_placeholderScreen_to_onboardingScreen"
```

Comment:
Can we rename this to `action_placeholderScreen_to_onboardingScreenOld`, please?